---
layout: post
title:  "Topic :: JSM2019"
date:   2019-07-31
categories: statistics topic
permalink: statistics
tags: statistics
use_math: true

# author
author: Kipoong Kim
---

<!-- more -->

## Joint Statistical Meeting 2019



### Sun, Jul. 28

#### Advances in Variable Selection—Contributed

#### Section on Statistical Learning and Data Science
Chair(s): Joanne C Beer, University of Pennsylvania

- 2:05 p.m. Simultaneous Confidence Regions for Coefficients in
  High-Dimensional Linear Models—Xiaorui Zhu,
  University of Cincinnati; Peng Wang, University of
  Cincinnati; Yichen Qin, University of Cincinnati

- 2:20 p.m. Functional Variable Selection with Correlated Functional
  Covariates and Longitudinal Responses—Rebecca
  North, NCSU Statistics; Jonathan Stallrich, North Carolina
  State University; Ana-Maria Staicu, North Carolina State
  University; Helen Huang, NCSU Biomedical Engineering;
  Dustin Crouch, University of Tennessee, Knoxville;
  Mechanical, Aerospace, and Biomedical Engineering

- 2:35 p.m. Feature Selection in Large Data with Heteroscedastic
  Errors—Yiying Fan, Cleveland State University

- 2:50 p.m. A New Information Criterion for Model Selection—
  Jie Ding, University of Minnesota; Vahid Tarokh, Duke
  University; Yuhong Yang, University of Minnesota

- 3:05 p.m. Floor Discussion

- **Summaries**
  **(1) They use the bootstrap method to get simultaneous confidence interval, but that is a biased estimator. Finally they use the proposed method plus lasso or adaptive lasso**

  **(2) Robotic Prothesies**

  **(3) There have been studies about intra-tumor heterogeneity(clonal theory; Nowell, 1976). They consider three measures, purity, ploidy and CNVs, of tumor using hidden marcov model. Also they modify the odds ratio and copy numbers with tumor purity (+casual/non-causal tumor purity). They use EM algorithm to estimate the proportion of subclone (purity), and it is available in R package “subHMM”**

  **It can help the tumor-targeted drug therapy**

  **(4) They want to identify patients who can benefit from the immuno therapy**

  **(5) They used wilma (supervised) clustering which is proposed by Marcel and Buhlmann** 

  *<u>**Keywords: Intra-tumor heterogeneity (purity+ploidy+CNV), Mutation burden(sum of mutation counts at each bin)</u>***

  

#### Medallion Lecture I—Invited

#### IMS

#### Organizer(s): Rajen D Shah, University of Cambridge

#### Chair(s): Sonia Petrone, Universitá commerciale Luigi Bocconi

- 4:05 p.m. On Statistical Thinking in Deep Learning—Yee Whye
  Teh, University of Oxford
- 5:45 p.m. Floor Discussion



### Mon, Jul. 29

#### SPEED: New Methods in Statistical Genomics and Genetics Part 1—Contributed

#### Section on Statistics in Genomics and Genetics

#### Chair(s): Tianzhong Yang, The University of Minnesota Twin Cities

- 8:35 a.m. Comparing Performance of Gene Set Test Methods
  Using Biologically Relevant Simulated Data—Richard
  Lambert, Utah State University; John Stevens, Utah
  State University

- 8:40 a.m. A Bottom-Up Approach to Testing Hypotheses That
  Have a Branching Tree Dependence Structure, with
  False Discovery Rate Control—Yunxiao Li, Emory
  University; Yijuan Hu, Emory University; Glen Alan
  Satten, Centers for Disease Control and Prevention

- 8:45 a.m. A Generalized Multi-Response Permutation Procedure
  to Evaluate Associations of Multivariate Data with
  Quantitative and Censored-Event Time Variables—
  FStanley Pounds, St. Jude Children’s Research Hospital;
  Natasha Sahr, St. Jude’s Children’s Hospital; Xueyuan
  Cao, University of Tennessee Health Science Center

- 8:50 a.m. The Robust Kernel Association Test—Kara Martinez,
  North Carolina State University

- 8:55 a.m. Regularized Regression by Graph Propagation for
  Genomic Data Analysis—Han Yu, Roswell Park
  Comprehensive Cancer Center; Rachael Hageman Blair,
  the State University of New York at Buffalo

- 9:00 a.m. Assessing Exposure Effects on Gene Expression Using
  Inverse Probability Weighting and the Parametric
  G-Formula—Sarah Reifeis, University of North
  Carolina at Chapel Hill; Michael Hudgens, University
  of North Carolina at Chapel Hill; Michael Love, UNCChapel
  Hill; Karen Mohlke, University of North Carolina
  at Chapel Hill; Melissa Troester, University of North
  Carolina at Chapel Hill

- 9:05 a.m. Methods for Handling Correlated Covariates in
  Integrative Genomics Analysis—Lauren Spirko-Burns,
  Karthik Devarajan, Fox Chase Cancer Center; Camille
  Ragin, Fox Chase Cancer Center

- 9:10 a.m. OncoCast: An Improved Interface for Survival Analysis
  Using Genomic Data—Axel Martin, Memorial Sloan
  Kettering Cancer Center

- 9:15 a.m. Identifying Appropriate Probabilistic Models for Sparse
  Discrete Omics Data—Hani Aldirawi, UIC

- 9:20 a.m. Bayesian Inference for Reconstructing Intra-Tumor
  Phylogeny—Tingting of Zhai, University of Kentucky;
  Jinpeng of Liu, University of Kentucky; Chi of Wang,
  University of Kentucky

- 9:30 a.m. PasLINCS: Pathway Activity Signatures from LINCS
  L1000 Consensus Gene Signatures—Yan Ren,
  University of Cincinnati; Siva Sivaganesan, University
  of Cincinnati; Nicholas Clark, University of Cincinnati;
  David Plas, University of Cincinnati; Mario Medvedovic,
  University of Cincinnati

- 9:35 a.m. Efficient Estimation of Ancestry Proportions Using
  Genotype Frequencies—Jordan Hall, University
  of Colorado Denver; Megan Sorenson, University of
  Colorado Denver; Ryan Scherenberg, ; Alexandria
  Ronco, University of Colorado Denver; Yinfei Wu,
  University of Colorado Denver; James Vance, University
  of Colorado Denver; Jinyan Lyu, University of Colorado
  Denver; Christopher Gignoux, University of Colorado
  Denver; Audrey E Hendricks, University of Colorado
  Denver

- 9:40 a.m. Likelihood Based Mixture Modeling of Genetic
  Regulatory Networks—David S. Burton, University of
  Rochester Biostatistics; Matthew N McCall, University
  of Rochester Medical Center

- 9:45 a.m. Selection of Genesets from a Cox Model with Higher-
  Order Interaction of Covariate Genes—Delong Liu,
  NHLBI/NIH; Colin O. Wu, National Heart, Lung and
  Blood Institute, National Institutes of Health; Beth
  Kozel, NHLBI/NIH; Neal Young, NHLBI/NIH

- 9:50 a.m. A Powerful and Versatile Colocalization Test—
  FYangqing Deng, University of Minnesota

- 9:55 a.m. The Rabl Configuration Limits Topological
  Entanglement of Chromosomes in Budding Yeast—
  FMaxime Pouokam, UC Davis Statistics Club

- 10:00 a.m. OASW Clustering—Fatima Batool,

- 10:05 a.m. Comparing Methods for Familial Relationship
  Inference in Populations with Complex Demographic
  History—Daniel Yorgov, Purdue University Fort
  Wayne
  10:10 a.m. On Simulating Ultra High-Dimensional Multivariate
  Data—Alfred Schissler, University of Nevada, Reno

- 10:15 a.m. Control Confounding by Familial Relatedness in
  Genome-Wide Association Studies—Annie J Lee,
  Columbia University; Donglin Zeng, UNC Chapel
  Hill; Badri N Varadarajan, Columbia University; Karen
  Marder, Columbia University ; Yuanjia Wang, Columbia
  University

- **Summaries**


  Keywords: Texanomic hierarchy in microbiome data



#### Frontiers of Statistical Genetics: Genomics, Transcriptomics, and PheWAS—Invited

#### WNAR, Section on Statistics in Genomics and Genetics, Biometrics Section

#### Organizer(s): Wei Sun, Fred Hutchinson Cancer Research Center
Chair(s): Wei Sun, Fred Hutchinson Cancer Research Center

- 10:35 a.m. Weighted Hypothesis Testing Accounting for Correlated
  Predictors—Li Hsu, Fred Hutchinson Cancer Research
  Center, USA

- 11:00 a.m. Scalable and Accurate Association Analysis for Big
  Biobank Data—Seunggeun Lee, University of
  Michigan

- 11:25 a.m. Mediation Analyzes for Dissecting the Role of DNA
  Methylation in Epidemiologic Studies—James Dai,
  Fred Hutchinson Cancer Research Center

- 11:50 a.m. Predictive Modeling of Transcriptomics in Ancestrally
  Diverse Populations—Timothy Thornton, University
  of Washington; Anya Mikhaylova, Universtiy of
  Washington

- 12:15 p.m. Floor Discussion

- **Summaries**
  The existing methods for analysis of sparse discrete data are ZIP, ZINB, PH and NBH. Since Kolmogorov-Smirnov test is available for continuous data, they used discrete KS test and estimate the parameters to be known using permutation.

  Keywords: Expression~SNP+CNV+Methyl+error, Sparse discrete data



#### Novel Statistical Methods for High-Dimensional Microbiome and Metagenomics Data Analysis—Topic Contributed
Section on Statistics in Epidemiology, Section on Statistics in Genomics and Genetics, Biometrics Section
Organizer(s): Chan Wang, Division of Biostatistics, NYU School of Medicine
Chair(s): Jiyuan Hu, New York Unversity School of Medicine

- 2:05 p.m. Analyzing Matched Sets of Microbiome Data Using LDM—Yijuan Hu, Emory University; Zhengyi Zhu,
  Emory University; Caroline Mitchell, Vincent Center for Reproductive Biology, Massachusetts General Hospital,
  Harvard Medical S; Glen Alan Satten, Centers for Disease Control and Prevention
- 2:25 p.m. Association Testing and Feature Selection for Microbiome and Host Genomics—Anna Plantinga,
  Williams College; Michael C. Wu, Fred Hutchinson Cancer Research Center
- 2:45 p.m. Multivariable Association in Population-Scale Meta’omic Surveys—Himel Mallick, Merck & Co., Inc.; Timothy Tickle, Broad Institute; Lauren McIver, Harvard University; Gholamali Rahnavard, Broad Institute;
  Long Nguyen, Massachusetts General Hospital; George Weingart, Harvard University; Siyuan Ma,
  Harvard University; Boyu Ren, Harvard University; Emma Schwager, Harvard University; Ayshwarya
  Subramanian, Broad Institute; Joseph Paulson, Genentech; Eric A. Franzosa, Harvard University;
  Hector Corrada Bravo, University of Maryland; Curtis Huttenhower, Harvard University
- 3:05 p.m. Robust Regression for Microbiome Data Analysis— FAditya Mishra, Flatiron Institute; Christian Lorenz
  Mueller, Flatiron Institute, Simons Foundation
- 3:25 p.m. Estimating and Testing the Microbial Causal Mediation Effect with High-Dimensional and Compositional
  Microbiome Data—Chan Wang, Division of Biostatistics, NYU School of Medicine; Jiyuan Hu, New
  York Unversity School of Medicine; Martin Blaser, New York University School of Medicine and Rutgers
  University; Huilin Li, NYU School of Medicine 
- 3:45 p.m. Floor Discussion



#### Bioinformatics: Accomplishments and Challenges—Invited
Caucus for Women in Statistics, Section on Statistics in Genomics
and Genetics, Section on Statistical Learning and Data Science
Organizer(s): Nusrat Jahan, James Madison University
Chair(s): Nusrat Jahan, James Madison University



- 8:35 a.m. Optimal Permutation Recovery and Estimation of Bacterial Growth Dynamics—Hongzhe Li, University
  of Pennsylvania
- 9:00 a.m. Estimating Somatic Variant Richness in the Cancer
  Genome—Ronglai Shen, Memorial Sloan-Kettering
  Cancer Center; Saptarshi Chakraborty, Memorial Sloan-
  Kettering Cancer Center; Colin Begg, Memorial Sloan
  Kettering Cancer Center
- 9:25 a.m. Integrative Network Modeling Approaches to Precision
  Cancer Medicine—Kim-Anh Do, University of Texas
  M.D. Anderson Cancer Center 
- 9:50 a.m. Statistical Inference of Chromatin 3D Structures from
  DNA Methylation Data—Shili Lin, The Ohio State
  University
- 10:15 a.m. Floor Discussion





#### Wald Lecture II—Invited

#### IMS
Organizer(s): Piotr Fryzlewicz, London School of Economics
Chair(s): Xihong Lin, Harvard

- 2:05 p.m. Wald II: Statistical Learning with Sparsity—Trevor J
  Hastie, Stanford University
- 3:05 p.m. Disc: Rahul Mazumder, MIT
- 3:25 p.m. Disc: William Fithian, University of California at Berkeley
- 3:45 p.m. Floor Discussion